lib/ood_core/job/adapters/linux_host/templates/script_wrapper.erb.sh

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
-hostname
+SSH_HOSTS=(<%= ssh_hosts.join(' ').to_s %>)
+hostnames=`hostname -A`
+for host in $SSH_HOSTS
+do
+    if [[ " ${hostnames[@]} " =~ " ${host} " ]]; then
+        hostname=$host
+    fi
+done
+
+if [ -z "$hostname" ]; then
+    echo >&2 "ERROR: The specified host is not in the list of ssh hosts configured in this cluster config [${SSH_HOSTS[@]}]. The specified hosts determined by running 'hostname -A' [${hostnames[@]}] on the target host must match one of the configured ssh hosts" 
```

Comment:
I wonder if this can't be worded a little simpler? Something like `ERROR: Can't start a job on '$(hostname -A)' because it's not like any hostname configured in ssh_hosts [${SSH_HOSTS[@]}]`.

Reply:
It definitely can be. Not sure whether this issue is being combined with https://github.com/OSC/ood_core/issues/190. The error message was taken directly from that issue.

Reply:
It seems like this should fix #190 because it'll trap the most common case for this adapter being in a bad state.

Yea I think the key here is "Something that explains the problem and suggests the solution". I would add "in plain words". 

How about this:
`ERROR: Can't start a job on '$(hostname -A)' because it's not like any hostname configured in ssh_hosts [${SSH_HOSTS[@]}]. The output of 'hostname -A' must match an entry in ssh_hosts from the cluster configuration.`

Also, you can add newlines in there too. As 1 line you have to scroll pretty far to the right.